addons/visibility/functions/fnc_draw.sqf

```diff
@@ -15,24 +15,28 @@
  * Public: No
  */
 
+if (call EFUNC(common,isInScreenshotMode)) exitWith {};
+
 // The cursor position in the world
 private _pos = AGLtoASL screenToWorld getMousePosition;
 private _intersections = lineIntersectsSurfaces [getPosASL curatorCamera, _pos];
 if !(_intersections isEqualTo []) then {
-    _pos = ((_intersections select 0) select 0);
+    _pos = _intersections select 0 select 0;
 };
 
-// check 1.5 above the cursor to prevent a small object on the terrain blocking the view
+// Check 1.5 above the cursor to prevent a small object on the terrain blocking the view
 private _posHigh = _pos vectorAdd [0, 0, 1.5];
 private _draw = false;
 {
-    if (side _x != sideLogic && {(((_x getRelDir _posHigh) + 90) mod 360) < 180}) then {
+    if (_x != player && {side _x != sideLogic} && {((_x getRelDir _posHigh) + 90) mod 360 < 180}) then {
```

Comment:
Seems like a lot of stuff is going in this singular if statement, maybe a comment is in order?

Reply:
@mharis001 Do you think a comment is necessary? I don't think it is,

Reply:
Doesn't hurt to add one.